awscli/customizations/configure.py

```diff
@@ -302,6 +303,9 @@ def _run_main(self, args, parsed_globals):
         region = self._lookup_config('region')
         self._display_config_value(region, 'region')
 
+        ca_bundle = self._lookup_config('ca_bundle')
```

Comment:
I'd prefer not to have this in the `aws configure` prompts.

We specifically ask a subset of config questions so that a new user can get up and running quickly by answering a small amount of questions.

I'd say configuring a custom ca_bundle is not in the common case for getting started.  Users can still run `aws configure set ca_bundle /foo` to set this programatically.